docs/products/kafka/concepts/acl.md

```diff
@@ -1,116 +1,216 @@
 
-## ACL structure
+Aiven supports two ACL models:
 
-The ACL consists of **ACL entries**. An ACL entry is defined as the
-combination of:
+- **Aiven ACLs**: Simplified topic-level access control with basic permissions and
+  wildcard support.
+- **Kafka-native ACLs**: Advanced resource-level access control with fine-grained
+  permissions, including `ALLOW` and `DENY` rules.
 
--   the username
--   the permission given to the user
--   the associated topics
+## Aiven ACL capabilities
 
-The username portion of the ACL entry can be an Apache Kafka® service
-user name, or a string containing wildcards, which can match multiple
-users. Similarly, the topic portion can be a single Apache Kafka® topic
-name or can use a wildcard pattern. The permission is one of `read`,
-`write`, `readwrite` and `admin`.
+Aiven ACLs provide a simpler way to manage access to Apache Kafka topics and Schema
+Registry resources. It is ideal for use cases requiring basic permissions and wildcard
+support.
 
-The wildcards supported are:
+- **Permissions**: Allow `read`, `write`, `readwrite`, or `admin` permissions to be
+  assigned to specific users or groups at the topic level.
+- **Wildcard support**: Simplifies access control by allowing patterns for usernames
+  and resource names, such as `logs-*` for topics or `user?` for usernames.
+- **User-specific or group-level permissions**: Provides precise access control by
+  applying permissions to specific users or groups. For instance, `username: analyst*`
+  applies to all usernames starting with "analyst."
 
--   `?` matching a single character (equivalent to regular expression
-    `.`)
--   `*` matching zero or more characters (equivalent to regular
-    expression `(.*)`)
+:::note
+By default, a user named `avnadmin` is created with `admin` permissions for all
+topics. If you are using the Aiven Terraform Provider and want to prevent this behavior,
+set `default_acl: false` in your resource configuration.
+:::
 
-The wildcards can be combined for more complex pattern matching:
+### ACL Structure
 
--   `?*` matches a single character, and zero or more other
-    characters (equivalent to regular expression `(.+)`)
+An Aiven ACL entry consists of the following elements:
 
-Aiven for Apache Kafka® evaluates each topic access against the ACL
-entries. If it finds a matching ACL entry, access is granted. If no
-entry matches, access is denied. The order of the ACL entries is
-irrelevant.
+- **Username**: The Aiven for Apache Kafka service username or a wildcard pattern.
+- **Permission**: One of `read`, `write`, `readwrite`, or `admin`.
+- **Associated topics**: Specific Kafka topics or wildcard patterns.
 
-Examples:
+Aiven checks ACL entries and grants access if there's a match. If no entry matches,
+access is denied. The order of ACL entries does not
+influence access evaluation.
 
```

Comment:
Maybe add here a note that Aiven ACL also gives implicitly access to all consumer groups?

Reply:
Added a note.